packages/atlas/src/hooks/useVideoCategoriesWithCounter.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react'
+
+import { useQnCategories } from '@/api/hooks/categories'
+import { displayCategories } from '@/config/categories'
+
+export const useVideoCategoriesWithCounter = () => {
```

Comment:
I think we could rename this to `useVideoDisplayCategoriesWithCounter` so it's clear those are display categories?